src/main/java/org/opentripplanner/standalone/config/ConfigLoader.java

```diff
@@ -32,6 +34,7 @@
     private static final String OTP_CONFIG_FILENAME = "otp-config.json";
     private static final String BUILD_CONFIG_FILENAME = "build-config.json";
     private static final String ROUTER_CONFIG_FILENAME = "router-config.json";
+    private static final String DATA_OVERLAY_CONFIG_FILENAME = "data-settings.json";
```

Comment:
I wonder if `data-settings.json` is a descriptive enough name. What about `data-overlay-config.json` or just `overlay-config.json`?

Reply:
good point, we will rename it to data-overlay-config.json so that it is consistent with the plugin name